src/zabapgit_view_repo.prog.abap

```diff
@@ -317,9 +318,15 @@ CLASS lcl_gui_view_repo_content IMPLEMENTATION.
                          iv_act = |{ gc_action-repo_refresh_checksums }?{ lv_key }| ).
     lo_tb_advanced->add( iv_txt = 'Remove'
                          iv_act = |{ gc_action-repo_remove }?{ lv_key }| ).
+
+    CLEAR lv_crossout.
+    IF mo_repo->is_write_protected( ) = abap_true
+        OR lcl_auth=>is_allowed( gc_activities-uninstall ) = abap_false.
+      lv_crossout = gc_html_opt-crossout.
+    ENDIF.
     lo_tb_advanced->add( iv_txt = 'Uninstall'
                          iv_act = |{ gc_action-repo_purge }?{ lv_key }|
-                         iv_opt = lv_wp_opt ).
+                         iv_opt = lv_crossout ).
 
```

Comment:
... menu stuff starts to become messy. I guess some refactoring should be done soon :) ...